airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/spec.json

```diff
@@ -0,0 +1,169 @@
+                             {
```

Comment:
what is the reason to roll back to spec.json ?

Reply:
See previous comment on "oneOf" type structure

Reply:
Waiting on the merge of PR [8485](https://github.com/airbytehq/airbyte/pull/8485) to implement a pydantic `oneOf` type structure.